apt-sources/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "apt-sources"
+authors = ["Michał Fita <4925040+michalfita@users.noreply.github.com>"]
+edition = "2021"
+version = "0.1.40"
+license = "Apache-2.0"
+description = "A parser for APT source files (package repositories specification)"
+repository = { workspace = true }
+homepage = { workspace = true }
+keywords = ["debian", "deb822", "rfc822", "lossless", "edit"]
```

Comment:
The current crate doesn't allow format-preserving editing, so perhaps drop the lossless and edit keywords - maybe add "apt"?